app/models/mail_handler.rb

```diff
@@ -548,9 +561,26 @@ def update_work_package(work_package)
 
   def collect_wp_attributes_from_email_on_update(work_package)
     attributes = issue_attributes_from_keywords(work_package)
-    attributes
+    addresses = [email.to, email.cc].flatten.compact.uniq.map { |a| a.strip.downcase }
+    if addresses.any?(/\+(\d+)/) == true
+      unless email.cc.nil?
+        cc_addresses = email.cc.join(', ')
+      else
+        cc_addresses = ''
+      end
+      attributes.merge(
+        'custom_field_values' => custom_field_values_from_keywords(work_package),
+        'journal_notes' => "Subject: #{email.subject.to_s.chomp[0, 255]}
+          To: #{email.to.join(', ')}
+          CC: #{cc_addresses}
+          Date: #{email.date.strftime('%d/%m/%y')}
+          Body:
+          #{cleaned_up_text_body}")
```

Comment:
The header is not i18ned. A french person might want to have e.g. `Body` translated.